### PR TITLE
fix: include `libpq`, `libcrypto-3`, `libssl-3` DLLs in the installer

### PR DESCRIPTION
**Pull request by harshdoesdev**
We are including these dlls which are required to make some PostgreSQL related dependencies work on Windows systems.

These are:
- https://www.dllme.com/dll/files/libpq
- https://www.dllme.com/dll/files/libssl-3-x64
- https://www.dllme.com/dll/files/libcrypto-3-x64

### CODE REVIEW

**Comment by amitu** [comment]
@harshdoesdev in the .github/lib repo add a README, with instruction on where you got the libraries and how to update them.

**Comment by harshdoesdev** [comment]
> Reference in new issue

@amitu I have added a README in the .github/lib directory with links and versions of the DLLs.

File: .github/lib/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+These DLLs are required for some PostgreSQL related dependencies on Windows. these are being packaged in the the installer and will be extracted in the fastn's installation directory on Windows.
+
+The DLLs are for PostgreSQL v16 but can also work with version 14. But in future this may change then these DLLs will have to be updated to their latest version.
+
+Downloaded from:
+- https://www.dllme.com/dll/files/libpq
+- https://www.dllme.com/dll/files/libssl-3-x64
+- https://www.dllme.com/dll/files/libcrypto-3-x64
+
+These DLLs are included in the install.nsi script in the installer and uninstall sections.
```
Comment (amitu) [inline .github/lib/README.md]: If the dll files are fixed I would appreciate if you created a bash script which does a curl. Also note down the hash of each dll.

File: .github/lib/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ## libpq, libcrypto-3, libssl-3
 
-These DLLs are required for some PostgreSQL related dependencies on Windows. these are being packaged in the the installer and will be extracted in the fastn's installation directory on Windows.
+These DLLs are required for certain PostgreSQL-related dependencies on Windows. They are being packaged in the installer and will be extracted in the fastn's installation directory on Windows.
```
Comment (amitu) [inline .github/lib/README.md:3]: These lines are too long. When looking at a PR in Github, they should not wrap like this.

**Pull request merged**